templates/include/telemetry/environment.1.schema.json

```diff
@@ -230,12 +228,12 @@
         ]
     },
     "experiments": {
+      "properties": {
```

Comment:
I believe this section was right before - we expect the structure:
```
"experiments": {
  "some_experiment_id": { "branch": "A" }
}
```
where `"some_experiment_id"` would be the additionalProperties, each with a structure with one property called `"branch"`.